.github/workflows/ort.yml

```diff
@@ -76,15 +76,11 @@ jobs:
               with: 
                   repository: "oss-review-toolkit/ort"
                   path: "./ort"
-                  ref: "20.1.0"
+                  ref: "26.0.0"
                   submodules: recursive
 
-            # Temporary workaround until ORT releases a version newer than 22.1.0 addressing https://github.com/oss-review-toolkit/ort/issues/8571.
-            # Upon resolution, remove this section and install the updated version instead of "20.1.0".
             - name: Install Rust toolchain
-              uses: dtolnay/rust-toolchain@1.76 
-              with:
-                targets: ${{ inputs.target }}
+              uses: dtolnay/rust-toolchain@1.78 
```

Comment:
Add github-token, otherwise at some point we start getting limit acceded errors.

Reply:
github-token doesn't apply to the rust-toolchain action:
https://github.com/dtolnay/rust-toolchain